src/components/vm/disks/diskAdd.tsx

```diff
@@ -337,13 +455,13 @@ const CustomPath = ({ idPrefix, onValueChanged, device, validationFailed, hideDe
                 <FileAutoComplete
                     id={`${idPrefix}-file-autocomplete`}
                     placeholder={_("Path to file on host's file system")}
-                    onChange={value => debounce(250, onValueChanged("file", value))}
+                    onChange={(value: string) => onValueChanged("file", value)}
```

Comment:
Please move this and the removal of the import to a separate commit, with a good justification.  This looks wrong at first sight?

Reply:
Done.